include/l3.h

```diff
@@ -84,4 +93,11 @@ void l3__log_simple(const uint32_t loc, const char *msg,
 #ifdef __cplusplus
 extern "C"
 #endif
-void l3_log_fast(const char *msg);
+
+#ifdef L3_LOC_ENABLED
+void l3__log_fast(const loc_t loc, const char *msg,
+                  const uint64_t arg1, const uint64_t arg2);
+#else
+void l3__log_fast(const uint32_t loc, const char *msg,
```

Comment:
Minor change done after you got the `.S` fast-logging interfaces working. Changed this to `const`.

Same on L98, above.